### DISCUSSION TITLE
How to create maximized resizable main window?

### DISCUSSION THREAD

**Discussion by r-pankevicius** [tryphotino/photino.NET#185] [Q&A]
Hi,

In examples I see using absolute sizes for main window, like `.SetSize(new Size(2048, 1024))` in Angular Demo App.

How to create resizable main window win minimize/maximize buttons that is maximized to host window at the start?

**Comment by MikeYeager** [comment]
@r-pankevicius I believe you're looking for .SetMaximized(true)

**Comment by r-pankevicius** [comment]
Thanks @MikeYeager , it was close.

This code compiled in WinExe project type starts main window maximized, window has minimize/maximize, close and restore (?) buttons.
```csharp
static void Main(string[] args)
{
	var window = new PhotinoWindow()
		.SetTitle("Window title")
		.SetMaximized(true);

	window.StartString = "Hello";

	window.WaitForClose();
}
```
However pressing that "restore" button (middle one, I don't know how it's called exactly, the window becomes minimized but still on top:
![image](https://github.com/tryphotino/photino.NET/assets/1467109/9e20b75b-c09f-4691-8eb1-4f03d7876eb4)
It's not a default behaviour in Windows, normally it sets window size to some "not maximized, but not minimized" size.

Adding `.SetUseOsDefaultSize(true)` in "window builder" code doesn't help.

**Comment by MikeYeager** [comment]
@r-pankevicius I believe you have run across a known bug in Photino. We broke the OS default sizes when we added a new feature recently. It's on our radar. We recently figured out a build issue that had caused us grief for a few weeks and this week is the BUILD conference, so we'll likely get into this next week. Sorry for the inconvenience. I believe it you set a size and turn of use defaults, you can also set maximized to true and the window will work as expected, but you will have to specify the 'normal' size.

**Comment by r-pankevicius** [comment]
No worries, @MikeYeager : I've just started to migrate the app from Electron.NET to Photino, I expect it to take some time, so I have time to wait for a fix.

**Comment by r-pankevicius** [comment]
@MikeYeager I saw it is solved in 3.1.18
